conf/init/nginx_conf_create.py

```diff
@@ -123,6 +123,23 @@ def generate_rate_limiting_config(config):
         static_dir=STATIC_DIR,
     )
 
+def generate_hosted_http_base_config(config):
+    """
+  Generates hosted http base config from the app config
+  """
+    config = config or {}
+
+    feature_proxy_protocol = config.get("FEATURE_PROXY_PROTOCOL", False)
```

Comment:
I'd send in the proxy protocol value and use an `{% if ` in the template

Reply:
PR updated